README.md

```diff
@@ -16,8 +16,8 @@ We welcome everyone with the expertise in the field. If you are interested in co
 - [Available predictive models](#available-predictive-models)
   - [Absorption](#absorption)
   - [Metabolism](#metabolism)
-- [Personal use](#personal-use)
-  - [Requirements](#requirements)
+- [Usage](#usage)
+  - [Install](#install)
```

Comment:
Isn't it better to have Installation? We have nouns everywhere. I think one verb among nouns looks strange)